packages/oauth/oauth_server.js

```diff
@@ -342,7 +342,10 @@ var renderEndOfLoginResponse = function (options) {
     throw new Error('invalid loginStyle: ' + options.loginStyle);
   }
 
-  var result = template.replace(/##CONFIG##/, JSON.stringify(config));
+  var result = template.replace(/##CONFIG##/, JSON.stringify(config))
+    .replace(
+      /##ROOT_URL_PATH_PREFIX##/, __meteor_runtime_config__.ROOT_URL_PATH_PREFIX
```

Comment:
Fair question. It's fine either way but in the current implementation, no, it's not possible.

However, looking at it from a "this looks like a template" angle with consideration for future changes: I'd still say "never" for `##CONFIG##` and "it's highly unlikely" for `##ROOT_URL_PATH_PREFIX##`.  The chances of [this template](https://github.com/meteor/meteor/blob/devel/packages/oauth/end_of_redirect_response.html) becoming more complicated are slim-to-none as it's just meant to load the one `<script>` which performs a trivial function.  Any cause for repetition of that tag would have to be meditated and I'd probably refute. :wink: